### PR TITLE
Bug 1161493 - responseURL get property for response object in sdk/request

### PR DESCRIPTION
**Pull request by skipme**
Allow get modified URL in case of a redirection. Forwarded responseURL property of encapsulated xhr.

### CODE REVIEW

File: test/test-request.js [hunk 1]
```diff
@@ -92,6 +92,35 @@ exports.testStatus200 = function (assert, done) {
...
+
+  var req = Request({
+    url: "http://localhost:" + port + "/" + basename,
+    onComplete: function (response) {
+      assert.equal(this, req, "`this` should be request");
+      assert.equal(response.status, 200);
+      assert.equal(response.statusText, "OK");
+      assert.equal(response.headers["Content-Type"], "text/plain");
+      assert.equal(response.text, movedContent);
+      assert.equal(response.responseURL, movedUrl);
```
Comment (erikvold) [inline test/test-request.js]: this should be strict equal, to make sure `response.responseURL` is a string, or test that separately.

File: lib/sdk/request.js [hunk 2]
```diff
@@ -165,6 +165,7 @@ const Response = Class({
     response(this).request = request;
   },
   get text() response(this).request.responseText,
+  get responseURL() response(this).request.responseURL,
```
Comment (erikvold) [inline lib/sdk/request.js]: The other property names here appear to have removed the `response` prefix, so in order to follow that pattern this line should be:

    get url() response(this).request.responseURL

**Comment by skipme** [comment]
Thank you, looks better.

**Comment by erikvold** [comment]
Thanks!

**Pull request merged**